numpy/core/src/umath/loops.c.src

```diff
@@ -2189,6 +2215,15 @@ HALF_negative(char **args, npy_intp *dimensions, npy_intp *steps, void *NPY_UNUS
 }
 
 NPY_NO_EXPORT void
+HALF_positive(char **args, npy_intp *dimensions, npy_intp *steps, void *NPY_UNUSED(func))
+{
+    UNARY_LOOP {
+        const npy_half in1 = *(npy_half *)ip1;
+        *((npy_half *)op1) = in1;
+    }
```

Comment:
I realize that you're just copying `negative` here - but what's the heuristic for when to use `UNARY_LOOP_FAST` and when to just use `UNARY_LOOP`? @juliantaylor, I suspect you can shed some light here?